code/game/machinery/doppler_array.dm

```diff
@@ -85,8 +107,15 @@ GLOBAL_LIST_EMPTY(doppler_arrays)
 	else
 		for(var/message in messages)
 			say(message)
+	output_log = messages.Join(" ")

```

Comment:
What's the point of keeping the collated log if you don't show it anywhere.

Reply:
iirc it was a temporary list

Reply:
Not sure what you mean but if it's unused remove it.

Reply:
I had to look at the code, this was used for the purpose of adding to the master log that gets display. It's so I could join the messages of the doppler's output then add it at one element at a time in the master log list.

Reply:
This is still in and unused

Reply:
It is used, did you not see it's lazyadded to the list used to log each explosion?

Reply:
It still has no purpose you can add the messages directly, the variable is completely redundant.